Assets/MixedRealityToolkit/_Core/Utilities/Lines/DataProviders/RectangleLineDataProvider.cs

```diff
@@ -0,0 +1,157 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Internal.Utilities.Lines.DataProviders
+{
+    public class RectangleLineDataProvider : BaseMixedRealityLineDataProvider
+    {
+        [SerializeField]
+        private Vector3[] points = new Vector3[8];
```

Comment:
tooltips

Reply:
These points are not rendered in the inspector, as such they don't need a tooltip.